gossip/gossip.go

```diff
@@ -85,29 +85,42 @@ const (
 	// ttlNodeDescriptorGossip is time-to-live for node ID -> address.
 	ttlNodeDescriptorGossip = 0 * time.Second
 
-	// stallInterval is the default interval for checking whether the
-	// incoming and outgoing connections to the gossip network are
+	// defaultStallInterval is the default interval for checking whether
+	// the incoming and outgoing connections to the gossip network are
 	// insufficient to keep the network connected.
-	stallInterval = 1 * time.Second
+	defaultStallInterval = 1 * time.Second
 
-	// bootstrapInterval is the minimum time between successive
-	// bootstrapping attempts to avoid busy-looping trying to find
-	// the sentinel gossip info.
-	bootstrapInterval = 1 * time.Second
-)
+	// defaultBootstrapInterval is the minimum time between successive
+	// bootstrapping attempts to avoid busy-looping trying to find the
+	// sentinel gossip info.
+	defaultBootstrapInterval = 1 * time.Second
 
-var (
-	// cullInterval is the default interval for culling the least
-	// "useful" outgoing gossip connection to free up space for a
-	// more efficiently targeted connection to the most distant node.
+	// defaultCullInterval is the default interval for culling the least
+	// "useful" outgoing gossip connection to free up space for a more
+	// efficiently targeted connection to the most distant node.
 	//
 	// Note: this value is a var instead of const for testing.
-	cullInterval = 60 * time.Second
+	defaultCullInterval = 60 * time.Second
+)
 
+var (
 	// TestBootstrap is the default gossip bootstrap used for running tests.
 	TestBootstrap = []resolver.Resolver{}
 )
 
+// Persistence is an interface which allows the gossip instance
+// to read and write bootstrapping data to persistent storage
+// between instantiations.
+type Persistence interface {
```

Comment:
FWIW there's some precedent (in etcd/raft and multiraft) for calling this interface `Storage` instead of `Persistence`.